repository/pom.xml

```diff
@@ -679,8 +679,8 @@
             <artifactId>maven-failsafe-plugin</artifactId>
             <configuration>
               <systemPropertyVariables>
-                <total.mod.count>6</total.mod.count>
-                <total.wi.count>15</total.wi.count>
+                <total.mod.count>8</total.mod.count>
```

Comment:
Should it be 8? Not just 7?

Reply:
kafka-item was also missing (check #156 )

Reply:
@mareknovotny apart from -Pproductized, what else is needed to execute repository tests?

Reply:
I think module number increase is understandable (we've added the itest module here), I just don't understand the increase in total.wi.count. Tests locally are failing for me with:
```
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[WARNING] The expression ${artifactId} is deprecated. Please use ${project.artifactId} instead. @ 
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead. @ 
[ERROR] 'dependencies.dependency.version' for org.testcontainers:kafka:jar is missing. @ line 130, column 17
[ERROR] 'dependencies.dependency.version' for org.testcontainers:toxiproxy:jar is missing. @ line 135, column 17
```
Is it possible that there needs to be done something around dependency management after the move to this repo?

Reply:
@jstastny-cz it's referenced in the first comment: [droolsjbpm-build-bootstrap#1482](https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1482)

Reply:
@jstastny-cz kept wi.count in 16, thanks!